acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+        self.base_url = base_url.rstrip("/") + "/"
+        self.timeout = timeout
+        self.session = aiohttp.ClientSession()
+
+    async def __aenter__(self):
+        return self
+
+    async def __aexit__(
+        self,
+        exc_type: type[BaseException] | None,
+        exc_val: BaseException | None,
+        exc_tb: Any | None,
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `exc_tb`

(ANN401)